var/spack/repos/builtin/packages/dyninst/package.py

```diff
@@ -20,78 +20,99 @@ class Dyninst(Package):
     version('9.2.0', tag='v9.2.0')
     version('9.1.0', tag='v9.1.0')
     version('8.2.1', tag='v8.2.1')
-    version('8.1.2', tag='v8.1.2')
-    version('8.1.1', tag='v8.1.1')
+
+    variant('openmp', default=True,
+            description='Enable OpenMP support for ParseAPI '
+            '(version 10.0.0 or later)')
+
+    variant('static', default=False,
+            description='Build static libraries')
 
     variant('stat_dysect', default=False,
-            description="patch for STAT's DySectAPI")
-
-    # Dyninst depends on libelf and libdwarf prior to @9.3.0
-    # Dyninst depends on elfutils and libdwarf from @9.3.0 to but
-    # not including @develop
-    # Dyninst depends on elfutils and elfutils libdw from @develop forward
-    # elf@0 is an abstaction for libelf
-    # elf@1 is an abstaction for elfutils
-    depends_on("elf@0", type='link', when='@:9.2.99')
-    # The sorting algorithm puts numbered releases as newer than alphabetic
-    # releases, but spack has special logic in place to ensure that
-    # develop is considered newer than all other releases.
-    # So, develop is included in the elf@1 line below.
-    depends_on("elf@1", type='link', when='@9.3.0:')
-    depends_on("libdwarf", when='@:9')
-    depends_on("boost@1.42:")
+            description="Patch for STAT's DySectAPI")
+
+    boost_libs = '+atomic+chrono+date_time+filesystem+system+thread+timer'
+
+    depends_on('boost@1.61.0:' + boost_libs)
```

Comment:
I would simply append the variants to this line instead of storing them in a variable name. Spack doesn't check `depends_on` for line length anyway.

Reply:
Actually, it wasn't for line length that I wrote it this way.
I simply wanted to keep the list of libraries separate.  So you
could add or remove one without touching the depends_on() line.
Ok?